x/lockup/abci.go

```diff
@@ -20,5 +20,6 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) []abci.ValidatorUpdate {
 		return []abci.ValidatorUpdate{}
 	}
 	k.WithdrawAllMaturedLocks(ctx)
+	k.DeleteAllMaturedSyntheticLocks(ctx)
```

Comment:
Does it make sense to have `DeleteAllMaturedSyntheticLocks` just be called within `WithdrawAllMaturedLocks`?

Reply:
I was thinking of splitting the native locks and synthetic locks in the same level so that native locks does not care about synthetic locks.